public/app/plugins/datasource/tempo/SearchTraceQLEditor/TraceQLSearch.tsx

```diff
@@ -85,37 +88,47 @@ const TraceQLSearch = ({ datasource, query, onChange }: Props) => {
     fetchTags();
```

Comment:
Btw seems like this fetch in case somebody changes the data source before finishing can lead to out of order response and wrong data. Should probably be canceled, or just use `useAsync`.